tst/teststandard/weakptr.tst

```diff
@@ -77,4 +77,19 @@ false
 gap> Unbind(w[2]);
 gap> Print(w,"\n");
 WeakPointerObj( [ 1, , E(5), 311, , fail ] )
+gap> Immutable(w);
+[ 1,, E(5), 311,, fail ]
+gap> w;
+WeakPointerObj( [ 1, , E(5), 311, , fail ] )
+gap> MakeImmutable(w);
+[ 1,, E(5), 311,, fail ]
+gap> w;
+[ 1,, E(5), 311,, fail ]
+gap> w := WeakPointerObj([1,,2^40*10^10,,[2,3,4],fail,]);
+WeakPointerObj( [ 1, , 10995116277760000000000, , [ 2, 3, 4 ], fail ] )
+gap> MakeImmutable(w);
+[ 1,, 10995116277760000000000,, [ 2, 3, 4 ], fail ]
+gap> GASMAN("collect");
+gap> w;
+[ 1,, 10995116277760000000000,, [ 2, 3, 4 ], fail ]
```

Comment:
I am not sure what the point of the this last test is? OK, you change `w` from a `WeakPointerObj` to a regular (but immutable) list. And of course the garbage collection does not touch it. So what?
And I suspect that even if the `MakeImmutable` was replaced by `Immutable`, nothing would change in this test, because `last2` or so would reference the (non-weak) result of `Immutable`, preventing the GC from touching anything in `w`.

Perhaps at least comment could be added that explains what's going on here, resp. what is being tested?